### ISSUE TITLE
Bootcamps Dependabot Alerts 

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/bootcamp-ecommerce#1285]
[Alert Link](https://github.com/mitodl/bootcamp-ecommerce/security/dependabot)

**Comment by arslanashraf7** [comment]
Putting it on hold as the only remaining alert at this stage is [node-fetch](https://github.com/mitodl/bootcamp-ecommerce/security/dependabot/yarn.lock/node-fetch/open) and we are already working on a similar fix in another application [here](https://github.com/mitodl/mitxpro/pull/2288). So it would be a good idea to wait on that other PR and not work in parallel on similar alerts.

**Comment by arslanashraf7** [comment]
I'm closing this ticket since the scope of this ticket is too broad and dynamic. We can get dependabot alerts every now and then and it would eventually be impossible to complete the scope of this ticket. We should fix the alerts on the go.
